commit e5a9a35cb9c0d92d7c986cb3696fb794be100087
Author: Helmut Schaa <helmut.schaa@googlemail.com>
Date:   Fri Mar 5 17:44:22 2010 +0100

    rt2x00: remove KSEG1ADDR define from rt2x00soc.h
    
    Remove the KSEG1ADDR define from rt2x00soc.h as it redefines and covers the
    correct one from the arch/mips/include/asm/addrspace.h. Otherwise the driver
    oopses on the target platform (Ralink rt3050 board).
    
    Signed-off-by: Helmut Schaa <helmut.schaa@googlemail.com>
    Acked-by: Ivo van Doorn <IvDoorn@gmail.com>
    Acked-by: Gertjan van Wingerde <gwingerde@gmail.com>
    Signed-off-by: John W. Linville <linville@tuxdriver.com>

diff --git a/drivers/net/wireless/rt2x00/rt2x00soc.h b/drivers/net/wireless/rt2x00/rt2x00soc.h
index 4739edf..474cbfc 100644
--- a/drivers/net/wireless/rt2x00/rt2x00soc.h
+++ b/drivers/net/wireless/rt2x00/rt2x00soc.h
@@ -26,8 +26,6 @@
 #ifndef RT2X00SOC_H
 #define RT2X00SOC_H
 
-#define KSEG1ADDR(__ptr) __ptr
-
 /*
  * SoC driver handlers.
  */
